docs/apps/lnw/es2k/es2k-lnw-underlay-lag.md

```diff
@@ -359,11 +361,21 @@ The following configuration assumes that:
 #     ip link add bond0 type bond miimon 100 mode active-backup
 # For dynamic LAG:
 #     ip link add bond0 type bond miimon 100 mode 802.3ad
+
+# Note: Some Linux environments have experienced issues in LAG/LACP configuring and rule programming.
+# Adding a 5s delay between each command resolves this timing issue.
```

Comment:
```suggestion
# Adding a 5s delay between commands resolves this timing issue.
```
Grammar error. _Each_ refers to individual members of a group or set. You can't be "between" an individual object.

You could say "between each _pair_ of commands," but why bother? "Between commands" is simpler.